<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="Measure">
      <property name="text">
       <string>Medir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QGroupBox" name="Lluvia">
        <property name="title">
         <string>Lluvia</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>321</width>
           <height>73</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Precipitación</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LineLluvia"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLineEdit" name="MaxLluvia"/>
              </item>
              <item>
               <widget class="QLineEdit" name="MinLluvia"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Viento">
        <property name="title">
         <string>Viento</string>
        </property>
        <widget class="QWidget" name="layoutWidget1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>321</width>
           <height>112</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Velocidad</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LineVel"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>m /s</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLineEdit" name="MaxVel"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="MinVel"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Dirección</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LineDirec"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>°Deg</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLineEdit" name="MaxDirec"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="MinDirec"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="Luz">
        <property name="title">
         <string>Luz</string>
        </property>
        <widget class="QWidget" name="layoutWidget2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>321</width>
           <height>52</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Intensidad Luminica</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LineLuz"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Lumens</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLineEdit" name="MaxLuz"/>
              </item>
              <item>
               <widget class="QLineEdit" name="MinLuz"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="Ambiente">
      <property name="title">
       <string>Ambiente</string>
      </property>
      <widget class="LedIndicator" name="led_hume" native="true">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>380</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="LedIndicator" name="led_temp" native="true">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>130</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>321</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Temperatura</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineTemp"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>°C</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="MaxTemp"/>
            </item>
            <item>
             <widget class="QLineEdit" name="MinTemp"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>321</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Humedad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineHum"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="MaxHum"/>
            </item>
            <item>
             <widget class="QLineEdit" name="MinHum"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LedIndicator</class>
   <extends>QWidget</extends>
   <header location="global">ledindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
